demo/demo.js

```diff
@@ -7,36 +7,35 @@ window.addEventListener('message', function (event) {
   }
 })
 
-var EMBED_DOM_CLASSES = [
-  {
-    selector: '.typeform-widget',
-    attribute: 'data-url'
-  }, {
-    selector: '.typeform-share',
-    attribute: 'href'
-  }, {
-    selector: '.typeform-full',
-    attribute: 'src'
-  }
-]
-
-const addQueryStringToUrl = (url, queryString) => {
-  return /\?/.test(url) ? `${url}&${queryString}` : `${url}?${queryString}`
-}
-
-window.addEventListener('DOMContentLoaded', function () {
-  var queryString = window.location.search.substring(1)
-  if (queryString.length === 0) {
-    return
-  }
-
-  EMBED_DOM_CLASSES
-    .forEach(function (embed) {
-      var attribute = embed.attribute
-      var embeds = document.querySelectorAll(embed.selector)
-      embeds.forEach(function (embed) {
-        var embedAttr = embed.getAttribute(attribute)
-        embed.setAttribute(attribute, addQueryStringToUrl(embedAttr, queryString))
-      })
-    })
-})
+// var EMBED_DOM_CLASSES = [
+//   {
+//     selector: '.typeform-widget',
+//     attribute: 'data-url'
+//   }, {
+//     selector: '.typeform-share',
+//     attribute: 'href'
+//   }, {
+//     selector: '.typeform-full',
+//     attribute: 'src'
+//   }
+// ]
+//
+// const addQueryStringToUrl = (url, queryString) => {
+//   return /\?/.test(url) ? `${url}&${queryString}` : `${url}#${queryString}`
+// }
+// window.addEventListener('DOMContentLoaded', function () {
+//   var queryString = window.location.search.substring(1)
+//   if (queryString.length === 0) {
+//     return
+//   }
+//
+//   EMBED_DOM_CLASSES
+//     .forEach(function (embed) {
+//       var attribute = embed.attribute
+//       var embeds = document.querySelectorAll(embed.selector)
+//       embeds.forEach(function (embed) {
+//         var embedAttr = embed.getAttribute(attribute)
+//         embed.setAttribute(attribute, addQueryStringToUrl(embedAttr, queryString))
+//       })
+//     })
+// })
```

Comment:
Does it make sense to remove commented lines entirely?

Reply:
Yes, I will remove it!